nautobot_chatops/models.py

```diff
@@ -23,9 +23,7 @@
     COMMAND_TOKEN_COMMENT_HELP_TEXT,
     COMMAND_TOKEN_TOKEN_HELP_TEXT,
 )
-from .integrations.grafana.models import Dashboard as GrafanaDashboard
-from .integrations.grafana.models import Panel as GrafanaPanel
-from .integrations.grafana.models import PanelVariable as GrafanaPanelVariable
+from .integrations.grafana.models import GrafanaDashboard, GrafanaPanel, GrafanaPanelVariable
```

Comment:
Oh good, I am glad that we can get rid of the rename. I am not sure why this wasn't done when originally implemented.

Reply:
I had to leave it in for the diffsync code because the diffsync models are called `GrafanaDashboard` and `GrafanaPanel` and I didn't want to go down another rabbit hole trying to fix those...